server/etcdserver/api/membership/cluster.go

```diff
@@ -508,9 +508,11 @@ func (c *RaftCluster) PromoteMember(id types.ID, shouldApplyV3 ShouldApplyV3) {
 	c.Lock()
 	defer c.Unlock()
 
-	m := *(c.members[id])
-	m.RaftAttributes.IsLearner = false
-	mustUpdateMemberInStore(c.lg, c.v2store, &m)
+	if _, ok := c.members[id]; ok {
+		m := *(c.members[id])
+		m.RaftAttributes.IsLearner = false
+		mustUpdateMemberInStore(c.lg, c.v2store, &m)
+	}
```

Comment:
Should we log something in the error case?

Reply:
It isn't an error during bootstrap phase, because it needs some time for the v2store to catch up with v3store. But added a info log anyway, thx